addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/BoxHandler.java

```diff
@@ -225,16 +270,23 @@ private synchronized void onUpdate() {
      */
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        logger.debug("update {} with {}", channelUID, command);
+        logger.debug("command for {}: {}", channelUID, command);
+        if (command instanceof RefreshType) {
+            // TODO
```

Comment:
There is a TODO, what should be done and can it be completed before merging the PR?